src/components/Attachments/AttachmentCarousel/index.tsx

```diff
@@ -158,7 +159,7 @@ function AttachmentCarousel({report, reportActions, parentReportActions, source,
             }
 
             const nextIndex = page + deltaSlide;
-            const nextItem = attachments[nextIndex];
+            const nextItem = attachments.at(nextIndex);
```

Comment:
Same, I'm not sure that `nextIndex` can be negative in this case, but let's double-check it.